Src/FluentAssertions/Types/TypeSelectorAssertions.cs

```diff
@@ -292,6 +292,47 @@ public AndConstraint<TypeSelectorAssertions> NotBeDecoratedWithOrInherit<TAttrib
             return new AndConstraint<TypeSelectorAssertions>(this);
         }
 
+        /// <summary>
+        /// Asserts that the all <see cref="Type"/> are sealed classes
```

Comment:
`TypeSelectorAssertions` has singular/plural mixed up.
`PropertyInfoSelectorAssertions.cs` and `MethodInfoSelectorAssertions.cs` use plural form.

```suggestion
        /// Asserts that the selected types are sealed
```

Feel free to fix the summaries for the other methods in this class, otherwise I'll fix them later :)